beesdoo_website_shift_swap/views/exchange_templates.xml

```diff
@@ -318,9 +318,15 @@
                     />
                 </form>
             </p>
+            <p t-if="same_timeslot" class="text-center">
+                <a
+                    class="btn btn-primary"
+                    href="/my/shift/possible/shift"
+                >No shifts suits me</a>
```

Comment:
```suggestion
                >No shift suits me</a>
```

Reply:
I agree with that, I just copied the other same sentences because I was too lazy to change them all

Reply:
Fixed everywhere. :)

Reply:
Thanks !